src/api/comments.ts

```diff
@@ -0,0 +1,14 @@
+import { client } from '../utils/fetchClient';
+import { Comment } from '../types/Comment';
+
+export const getComments = (postId: number) => {
+  return client.get<Comment[]>(`/comments?postId=${postId}`);
+};
+
+export const removeComment = (commentId: number) => {
+  return client.delete(`/comments/${commentId}`);
+};
+
+export const createComment = (comment: Omit<Comment, 'id'>) => {
+  return client.post('/comments', comment);
+};
```

Comment:
since you'll need the received data from the server, it's good to add types
```ts
export const createComment = (comment: Omit<Comment, 'id'>): Promise<Comment> => {
  return client.post<Comment>('/comments', comment);
};